packages/salesforcedx-vscode-core/package.json

```diff
@@ -53,7 +53,8 @@
     "nyc": "^13",
     "sinon": "^7.3.1",
     "typescript": "3.1.6",
-    "vscode": "^1.1.36"
+    "vscode": "1.1.17",
+    "xmldom-sfdx-encoding": "0.1.29"
```

Comment:
This is the library I saw force-com-toolbelt using for xml parsing

Reply:
@AnanyaJha this should be moved to `dependencies` or it will not be bundled in the vsix and the extension will fail to activate.